### PR TITLE
Fix some layout issues in German translation

### PR DESCRIPTION
**Pull request by ReneSchimmelpfennig**

### CODE REVIEW

**Comment by Rem0o** [comment]
Please open a single big PR instead of individuals ones for every commit.

File: HomePage/HomePage.de.json [hunk 1]
```diff
@@ -27,13 +27,13 @@
   "RefreshSensors": "Sensorerkennung neu laden",
   "SaveConfig": "Konfiguration speichern",
   "SaveConfigAs": "Konfiguration speichern als ….",
-  "Settings": "Einstellungen",
+  "Settings": "Einst.",
   "Setup": "Einrichtung",
   "ShowHiddenCards": "Ausgeblendete Karten anzeigen",
   "Speeds": "Drehzahlen",
   "TemperatureUnitQuestion": "Um die Änderung der Temperatureinheit anzuwenden, wird die Konfiguration gespeichert und das Anwendungsfenster zurückgesetzt. Fortfahren?",
   "Theme": "Design",
```
Comment (Rem0o) [inline HomePage/HomePage.de.json:35]: Use \r\n to force a newline between "Tray" and "icons",  like so:

Tray\r\nIcons

**Comment by ReneSchimmelpfennig** [comment]
> Please open a single big PR instead of individuals ones for every commit.

Oops, I'm sorry, I'll do that from now on

**Review by ReneSchimmelpfennig — COMMENTED** [review]
Thanks, it's corrected now

**Pull request merged**